api/src/services/users.js

```diff
@@ -399,7 +407,22 @@ const deleteUser = id => {
   return Promise.all([ usersDbPromise, medicDbPromise ]);
 };
 
-const validatePassword = password => {
+/**
+ * Generates a complex enough password with a given length
+ * @param {Number} length
+ * @returns {string} - the generated password
+ */
+const generatePassword = (length = PASSWORD_MINIMUM_LENGTH) => {
+  const chars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789.,:!$-=';
+  let password;
+  do {
+    password = Array.from({ length }).map(() => _.sample(chars)).join('');
+  } while (passwordTester(password) < PASSWORD_MINIMUM_SCORE);
```

Comment:
If we're generating passwords nobody is ever going to use then let's go nuts on security, eg:
```suggestion
  } while (password.length < 20 && passwordTester(password) < PASSWORD_MINIMUM_SCORE);
```